### PR TITLE
Server manual

### LINKED ISSUE #1232: Create a Game Manual
**What should be documented? Is there something wrong in the documentation?**
With a new client manual soon to be published, we need to do a full documentation of the server and all its commands along with some recipes for usage and running your own.

**Do you have suggestions?**
Add to new `Manuals` section same as with client manual.

### PR DESCRIPTION
**Pull request by jwrober**
Potentially closes #1232

### CODE REVIEW

**Comment by jwrober** [comment]
https://longturn--1927.org.readthedocs.build/en/1927/Manuals/Server/index.html

File: docs/Manuals/Server/options.rst [hunk 1]
```diff
@@ -0,0 +1,1176 @@
...
+  thanks to the ``startpos`` setting, this setting controls how the start positions will be assigned to the
+  different players of the same team. Possible Values:
+
+    * Disabled (``DISABLED``): The start positions will be randomly assigned to players, regardless of teams.
+    * As close as possible (``CLOSEST``): Players will be placed as close as possible, regardless of
+      continents.
+    * On the same continent (``CONTINENT``): If possible, place all players of the same team onto the same
+      island/continent.
+    * Horizontal placement (``HORIZONTAL``): Players of the same team will be placed horizontally.
+    * Vertical placement (``VERTICAL``): Players of the same team will be placed vertically.
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:941]: Same as above, values and their names should be swapped

File: docs/Manuals/Server/options.rst [hunk 2]
```diff
@@ -0,0 +1,1176 @@
...
+    * ``WRAPY``: Wrap North-South
+    * ``ISO``: Isometric
+    * ``HEX``: Hexagonal
+
+``trade_revenue_style``
+  :strong:`Value (Min, Max)`: Classic Freeciv (``CLASSIC``)
+
+  :strong:`Description`: The formula for the trade a city receives from a traderoute. ``CLASSIC`` revenues
+  depend on distance and trade with multipliers for overseas and international routes. ``SIMPLE`` revenues are
+  proportional to the average trade of the two cities.
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1061]: Should be a list of values instead of a paragraph

File: docs/Manuals/Server/options.rst [hunk 3]
```diff
@@ -0,0 +1,1176 @@
...
+
+``trademindist``
+  :strong:`Value (Min, Max)`: 9 (1, 999)
+
+  :strong:`Description`: Minimum distance (tiles) for trade routes. In order for two cities in the same
+  civilization to establish a trade route, they must be at least this far apart on the map. For square grids,
+  the distance is calculated as *Manhattan distance*, that is, the sum of the displacements along the
+  :math:`x` and :math:`y` directions.
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1069]: This should probably mention hexagonal grids as well, since it mentions square grids

File: docs/Manuals/Server/options.rst [hunk 4]
```diff
@@ -0,0 +1,1176 @@
...
+  until all players have finished their turn, including disconnected players.
+
+``unitwaittime``
+  :strong:`Value (Min, Max)`: 0 (0, 8639999)
+
+  :strong:`Description`: Minimum time between unit actions over turn change. This setting gives the minimum
+  amount of time in seconds between unit moves and other significant actions (such as building cities) after a
+  turn change occurs. For example, if this setting is set to 20 and a unit moves 5 seconds before the turn
+  change, it will not be able to move or act in the next turn for at least 15 seconds. This value is limited
+  to a maximum value of two-thirds of ``timeout``. ``unitwaittime`` (aka UWT) is a tool to help reduce
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1117]: Do we have UWT in the glossary? We probably should, it's a fairly commonly used term, especially in LT games. This should then link to it.

File: docs/Manuals/Server/options.rst [hunk 5]
```diff
@@ -0,0 +1,1176 @@
...
+  newly-built units are subject to ``unitwaittime`` so that the moment the city production was last touched
+  counts as their last *action*. Also, getting captured/bribed counts as action for the victim.
+
+``unitwaittime_style``
+  :strong:`Value (Min, Max)`: empty value/not set
+
+  :strong:`Description`: This setting affects ``unitwaittime`` and effectively unused as it only has one option
+  to set:
+
+    * Activities (``ACTIVITIES``): Units moved less than ``unitwaittime`` seconds from turn change will not
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1133]: Option value seems pretty self-explanatory, the short value description seems redundant
```suggestion
    * ``ACTIVITIES``: Units moved less than ``unitwaittime`` seconds from turn change will not
```

File: docs/Manuals/Server/options.rst [hunk 6]
```diff
@@ -0,0 +1,1176 @@
...
+``victories``
+  :strong:`Value (Min, Max)`: Spacerace (``SPACERACE``) and Allied victory (``ALLIED``)
+
+  :strong:`Description`: What kinds of victories are possible for the game. This setting controls how a game
+  can be won. One can always win by conquering the entire planet, but other victory conditions can be enabled
+  or disabled:
+
+    * Spacerace (``SPACERACE``): Spaceship is built and travels to Alpha Centauri.
+    * Allied (``ALLIED``): After defeating enemies, all remaining players are allied.
+    * Culture (``CULTURE``): Player meets ruleset defined cultural domination criteria.
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1154]: The option values seem pretty self-explanatory, I don't think the short value descriptions are needed here
```suggestion
    * ``SPACERACE``: Spaceship is built and travels to Alpha Centauri.
    * ``ALLIED``: After defeating enemies, all remaining players are allied.
    * ``CULTURE``: Player meets ruleset defined cultural domination criteria.
```

File: docs/Manuals/Server/options.rst [hunk 7]
```diff
@@ -0,0 +1,1176 @@
...
+
+``ysize``
+  :strong:`Value (Min, Max)`: 64 (16, 128000)
+
+  :strong:`Description`: Map height in tiles. Defines the map height. For this option to take effect, the
+  "Map size definition" option (``mapsize``) must be set to "Width and height" (``XYSIZE``).
+
+  .. raw:: html
+
+        <p>&nbsp;</p>
```
Comment (daavko) [inline docs/Manuals/Server/options.rst:1176]: Why is this here? It doesn't appear in other files

**Review by daavko — COMMENTED** [review]
I'm sure I missed things, but at least it's something

File: docs/Manuals/Server/commands.rst [hunk 8]
```diff
@@ -0,0 +1,515 @@
...
+  random name will be assigned when the game begins. Until then the player will be known by a name derived
+  from its type. The ``ai type`` parameter can be used to select which :term:`AI` module will be used for the
+  created player. This requires that the respective module has been loaded or built in to the server. If the
+  game has already started, the new player will have no units or  cities. Also, if no free player slots are
+  available, the slot of a dead player can be reused (removing all record of that player from the running
+  game).
+
+``/away``
+  Toggles ``away`` mode for your nation. In away mode, the :term:`AI` will govern your nation but make only
+  minimal changes.
```
[the remark this answers is not in the document]
Comment (jwrober) [reply docs/Manuals/Server/commands.rst:178]: That is text straight from the server help. I have no idea what "minimal changes" mean either LoL

[the remark this answers is not in the document]
Comment (jwrober) [reply docs/Manuals/Server/commands.rst:178]: I added this

```
.. note::
  The term *minimal changes* is not well understood at this time. The server help does not provide more
  details. An enterprising enthusiast could read the :term:`AI` code to determine what the term means and
  provide more details. Any real player is not going to want the AI to run thier nation and will
  :ref:`delegate <server-command-delegate>` instead.
```

File: docs/Manuals/Server/options.rst (hunk 3 above)
Comment (jwrober) [reply docs/Manuals/Server/options.rst:1069]: is the math different for hex?

File: docs/Manuals/Server/options.rst (hunk 7 above)
Comment (jwrober) [reply docs/Manuals/Server/options.rst:1176]: the original file was a giant table and i added that to create some whitespace between the table and the buttons at the bottom of the page.
